04_git_strategies/README.md

```diff
@@ -9,3 +9,15 @@ Git is very flexible and can be tailored to many types of projects and teams. Th
 After this lesson you should:
 
 - Know four strategies for working with Git with your team in a more seamless fashion.
+
+## Pre-commit (0:06)
+
+![Pre-commit cover](../images/pre-commit-cover.png)
+
+[Link to video](https://www.youtube.com/watch?v=psjz6rwzMdk)
```

Comment:
**suggestion**: What do you think about moving the video to after the text? I think that the explanation you give below could motivate the students to pay closer attention to the video.

Reply:
Done!